radish34/api/src/baseline/core/integrations/messenger/sendMessage/acknowledge.js

```diff
@@ -1,8 +1,9 @@
 //
 export default (job, done) => {
   try {
+    const { data } = job;
     // Update Baseline to say that it's in motion
-    done(null, { message: 'Acknowledge done', jobData: job.data });
+    done(null, { message: 'Acknowledge done', jobData: data });
```

Comment:
I think if we just use the attr once, no need to destructure

Reply:
Did the changes and pushed the commit.